kyaml/filesys/fsondisk_test_windows.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package filesys
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sys/windows"
```

Comment:
I'm guessing this file is supposed to have a `// +build windows` tag?

Reply:
We don't need it in this case because the `_windows` in the filename already indicates this ;)